### PR TITLE
Kokonusswasser

### PR DESCRIPTION
**Pull request by jMyles**
Remove the DHT, and Run a node from the NuCypher CLI

### CODE REVIEW

File: cli/main.py [hunk 1]
```diff
@@ -58,15 +58,18 @@
...
+        # click.echo("Using configuration filepath {}".format(self.config_filepath))
+
+        self.operating_mode = operating_mode
+        self.simulation_mode = simulation_mode
 
         # Set operating and run modes
-        operating_modes = parse_running_modes(filepath=self.config_filepath)
-        self.operating_mode = operating_modes.get('mode', 'decentralized')
-        self.simulation_mode = operating_modes.get('simulation', False)
+        if use_config is True:
```
Comment (cygnusv) [inline cli/main.py]: Just nitpicking, but this expression seems redundant. Perhaps just `if use_config:` ?

Comment (cygnusv) [reply cli/main.py]: The same patterns seems to be repeating below.

File: cli/demos/finnegans-wake-demo.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# This is an example of Alice setting a Policy on the NuCypher network.
```
Comment (cygnusv) [inline cli/demos/finnegans-wake-demo.py]: So cool to see this in the CLI !! 👏

File: cli/main.py [hunk 3]
```diff
@@ -69,8 +84,9 @@ def __init__(self):
         sim_mode = 'simulation' if self.simulation_mode else 'live'
         click.echo("Running in {} {} mode".format(sim_mode, self.operating_mode))
 
-        self.payload = parse_nucypher_ini_config(filepath=self.config_filepath)
-        click.echo("Successfully parsed configuration file")
+        if use_config is True:
```
Comment (tuxxy) [inline cli/main.py]: Can this not be moved under the condition on L70?

Comment (KPrasch) [reply cli/main.py]: This is deprecated upstream, it was just a holdover for the hackathon.

Comment (tuxxy) [reply cli/main.py]: :+1:

File: cli/main.py (hunk 1 above)
Comment (KPrasch) [reply cli/main.py]: Deprecated upstream, but I generally adopted this pattern in case the user unintentionally passes a non-boolean type.

**Pull request merged**